native/src/components/__tests__/News.spec.tsx

```diff
@@ -0,0 +1,156 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import moment from 'moment'
+import React from 'react'
+import { Text } from 'react-native'
+import { ThemeProvider } from 'styled-components/native'
+
+import {
+  CityModel,
+  LOCAL_NEWS_TYPE,
+  LocalNewsModel,
+  LocalNewsType,
+  TU_NEWS_TYPE,
+  TunewsModel,
+  TuNewsType
+} from 'api-client'
+
+import buildConfig from '../../constants/buildConfig'
+import News from '../News'
+
+jest.mock('react-i18next')
+jest.mock('../../components/NativeHtml', () => ({ content }: { content: string }) => <Text>{content}</Text>)
+
+const news = [
```

Comment:
This way you don't have to use the `!` below
```suggestion
const news: [TunewsModel, TunewsModel]
```